tutorials/blockchains/testing/spawn-basic-chain.md

```diff
@@ -0,0 +1,91 @@
+```
+
+This configuration file defines a network with the following chains:
+
+- **relaychain** - with two nodes named `alice` and `bob` 
+- **parachain** - with a collator named `collator01` 
+
+Settings also defines a timeout of 120 seconds for the network to be ready.
+
+## Spawn the Network
+
+To spawn the network, run the following command:
+
+```bash
+zombienet -p native spawn spawn-a-basic-network.toml
+```
+
+This command will spawn the network defined in the `spawn-a-basic-network.toml` configuration file. The `-p native` flag specifies that the network will be spawned using the native provider.
+
+If successful, you will see the following output:
+
+--8<-- 'code/tutorials/blockchains/testing/spawn-basic-chain/spawn-network-terminal-01.html'
+
+!!! note 
+    If the IPs and ports aren't explicitly defined in the configuration file, they may change each time the network is started, causing the links provided in the output to differ from the example.
+
+## Interact with the Spawned Network
+
+After the network is launched, you can interact with it using [Polkadot.js Apps](https://polkadot.js.org/apps/){target=\_blank}. To do so, open your browser and use the provided links listed by the output as `Direct Link`.
+
+### Connect to the Nodes
+
+Use the [55308 port address](https://polkadot.js.org/apps/?rpc=ws://127.0.0.1:55308#explorer){target=\_blank} to interact with the same `alice` node used for this tutorial. Ports can change from spawn to spawn so be sure to locate the link in the output when spawning your own node to ensure you are accessing the correct port.
+
+If you want to interact with the nodes more programmatically, you can also use the [Polkadot.js API](https://polkadot.js.org/docs/api/){target=\_blank}. For example, the following code snippet shows how to connect to the `alice` node using the Polkadot.js API and log some information about the chain and node:
+
+```typescript
+--8<-- 'code/tutorials/blockchains/testing/spawn-basic-chain/connect-to-alice-01.js'
+```
+
+Both methods allow you to interact easily with the network and its nodes.
+
+### Check Metrics
+
+You can also check the metrics of the nodes by accessing the links provided in the output as `Prometheus Link`. [Prometheus](https://prometheus.io/){target=\_blank} is a monitoring and alerting toolkit that collects metrics from the nodes. By accessing the provided links, you can see the metrics of the nodes in a web interface. So, for example, the following image shows the Prometheus metrics for Bob's node from the Zombienet test:
+
+![](/images/tutorials/blockchains/testing/spawn-basic-chain/spawn-basic-network-01.webp)
+
+### Check Logs
+
+To check node logs, you can use the command listed in the output as `Log Cmd`. For instance, to check the logs of the `alice` node, you can open a new terminal and run the following command:
+
+```bash
+tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-SEzfCidQ1za4/alice.log
```

Comment:
This command is slightly different from the one you show in terminyl. I think this one here is the correct one because it ends in .log but I'll leave it up to you to confirm. 

This is the command shown in the terminyl snippet: 
tail -f /var/folders/f4/7rdt2m9d7j361dm453cpggbm0000gn/T/zombie-75a01b93c92d571f6198a67bcb380fcd_21724-2